packages/amplify-e2e-tests/src/__tests__/function.test.ts

```diff
@@ -316,3 +316,68 @@ describe('amplify add function with additional permissions', () => {
     expect(lambdaSource.includes('GRAPHQLAPIIDOUTPUT')).toBeTruthy();
   });
 });
+
+describe.only('amplify add/update/remove function based on schedule rule', () => {
```

Comment:
We need a lot more tests and test execution as well, udpate, delete flow, etc.

Reply:
added tests for add/update/delete